just-install.json

```diff
@@ -1471,6 +1471,23 @@
       },
       "version": "latest"
     },
+    "paint.net": {
+      "version": "4.1.6",
+      "installer": {
+        "kind": "custom",
+        "x86": "https://www.dotpdn.com/files/paint.net.{{.version}}.install.zip",
+        "options": {
+          "arguments": [
+            "{{.installer}}",
+            "/auto"
+          ],
+          "container": {
+            "kind": "zip",
+            "installer": "paint.net.4.1.6.install.exe"
```

Comment:
I'm not sure how much I like having the version hardcoded in the container argument. @lvillani?

Reply:
This really is unfortunate, but `{{.version}}` doesn't work here. In all honesty, it would be very helpful to have `{{.installer}}` and `{{.version}}` work everywhere and not just limited to a select few properties.

Reply:
I agree with @tlm-2501. I'm fine having it hard-coded for the time being, though.